WooCommerce/Classes/ViewRelated/Orders/Order Creation/CustomerSection/CustomerSelectorViewController.swift

```diff
@@ -76,7 +80,19 @@ private extension CustomerSelectorViewController {
         dismiss(animated: true)
     }
 
-    @objc func presentNewCustomerDetailsFlow() {}
+    @objc func presentNewCustomerDetailsFlow() {
+        let editOrderAddressForm = EditOrderAddressForm(dismiss: { [weak self] in
+                                                            self?.dismiss(animated: true, completion: {
+                                                                self?.dismiss(animated: true)
+                                                            })
```

Comment:
We seem to target the `CustomerSelectionViewController`'s dismiss with both `self.dismiss` callbacks, would this work instead?

```
        let editOrderAddressForm = EditOrderAddressForm(dismiss: { [weak self] in
            self?.dismiss(animated: true)
        }, showSearchButton: false, viewModel: addressFormViewModel)
```

If that's not the case, should we weakify self in the second completion handler as well?

Reply:
With only one dismiss we just dismissed the form view, we wanted both of them to be dismissed, but I weakified self in 797cf65426da3cde8618cd6160f6f4afce12461e